Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:2.7-alpine
 WORKDIR /src
 RUN pip install pipenv
 COPY Pipfile Pipfile.lock /src/
-RUN pipenv install --system --deploy
+RUN pipenv install --system --skip-lock --deploy
```

Comment:
I think `--deploy` only makes sense when we have a Pipfile.lock

Reply:
Fixed.